lsst/qserv
# Add support for NCSA PDAC deployment

* Add support for ssh server names
  different from hostname
* Refine node names management
* Add advanced configuration file
* Add parallel management scripts:
  - test ssh availability
  - create log directories on hosts
  - export initial MySQL/qserv from container
    to hosts
  - check Docker existence
  - restart Docker

Document NCSA cluster management
